tests/unit/specs/__snapshots__/App.spec.js.snap

```diff
@@ -10,7 +10,7 @@ exports[`App-root App render should render with loading 1`] = `
 
 exports[`App-root App render should render without loading markup 1`] = `
 <theme-provider-stub theme="default">
-  <loading-screen-stub>
+  <loading-screen-stub isloading="true">
```

Comment:
App render should render without loading -> isLoading true - is it ok? should we fix test?